### PR TITLE
Add endpoint for getting bridge payout transaction from evm tx hash

### LINKED ISSUE #10: Add Github Action for running Unit Tests

### PR DESCRIPTION
**Pull request by vectorman1**
**Detailed description**:

**Which issue(s) this PR fixes**:
Fixes #10

**Special notes for your reviewer**:

**Checklist**
- [ ] Documentation added
- [ ] Tests updated

### CODE REVIEW

File: app/services/utils/service.go [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+func (s *utilsService) ConvertEvmHashToBridgeTxId(txId string, chainId uint64) (*service.BridgeTxId, error) {
+	bridgeAbi, err := abi.JSON(strings.NewReader(router.RouterABI))
+	if err != nil {
+		return nil, err
+	}
+
+	burnHash := bridgeAbi.Events["Burn"].ID
+	lockHash := bridgeAbi.Events["Lock"].ID
+	burnERC721Hash := bridgeAbi.Events["BurnERC721"].ID
```
Comment (failfmi) [inline app/services/utils/service.go:54]: This can be done only once and stored in the struct/vars

File: app/services/utils/service.go [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+	if !ok {
+		return nil, errors.New("invalid chain id")
+	}
+
+	txHash := common.HexToHash(txId)
+	receipt, err := client.WaitForTransactionReceipt(txHash)
+	if err != nil {
+		return nil, errors.Wrap(err, "error while waiting for receipt")
+	}
+	s.log.Debugf("%#v", receipt)
```
Comment (failfmi) [inline app/services/utils/service.go:66]: Could we refactor this to something more readable?

File: app/services/utils/service.go [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+		case burnHash:
+			txIdWithLogIndex = fmt.Sprintf("%s-%d", txId, l.Index)
+			goto finish
+		case burnERC721Hash:
+			txIdWithLogIndex = fmt.Sprintf("%s-%d", txId, l.Index)
+			goto finish
+		case lockHash:
+			txIdWithLogIndex = fmt.Sprintf("%s-%d", txId, l.Index)
+			goto finish
+		}
```
Comment (failfmi) [inline app/services/utils/service.go:79]:
```
		switch l.Topics[0] {
		case burnHash:
		case burnERC721Hash:
		case lockHash:
			txIdWithLogIndex = fmt.Sprintf("%s-%d", txId, l.Index)
			goto finish
		}
```

File: app/services/utils/service_test.go [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+		80001: mocks.MEVMClient,
+	}, mocks.MBurnService)
+
+	assert.Equal(t, svc, actual)
+}
+
+func Test_ConvertEvmTxIdToHederaTxId_LockEvent(t *testing.T) {
+	setup()
+	mocks.MEVMClient.On("WaitForTransactionReceipt", evmTxHash).Return(&types.Receipt{
+		Logs: []*types.Log{
```
Comment (Psykepro) [inline app/services/utils/service_test.go:78]: You can move this on top as var and reuse it by just changing the last Log entry's hash.

File: app/services/utils/service_test.go [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+	actual, err := svc.ConvertEvmHashToBridgeTxId(evmTx, 80001)
+
+	assert.Nil(t, err)
+	assert.Equal(t, expectedResult, actual)
+}
+
+func Test_ConvertEvmTxIdToHederaTxId_BurnEvent(t *testing.T) {
+	setup()
+	mocks.MEVMClient.On("WaitForTransactionReceipt", evmTxHash).Return(&types.Receipt{
+		Logs: []*types.Log{
```
Comment (Psykepro) [inline app/services/utils/service_test.go:116]: You can move this on top as var and reuse it by just changing the last Log entry's hash.

File: app/services/utils/service_test.go [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+	actual, err := svc.ConvertEvmHashToBridgeTxId(evmTx, 80001)
+
+	assert.Nil(t, err)
+	assert.Equal(t, expectedResult, actual)
+}
+
+func Test_ConvertEvmTxIdToHederaTxId_BurnErc721Event(t *testing.T) {
+	setup()
+	mocks.MEVMClient.On("WaitForTransactionReceipt", evmTxHash).Return(&types.Receipt{
+		Logs: []*types.Log{
```
Comment (Psykepro) [inline app/services/utils/service_test.go:154]: You can move this on top as var and reuse it by just changing the last Log entry's hash.

File: app/services/utils/service_test.go (hunk 4 above)
Comment (svetlio8) [inline app/services/utils/service_test.go:78]: maybe we can have it at the beginning `var (...)` and re-used in the `Test_ConvertEvmTxIdToHederaTxId_LockEvent`, `Test_ConvertEvmTxIdToHederaTxId_BurnEvent`, and `Test_ConvertEvmTxIdToHederaTxId_BurnErc721Event`?

File: app/router/utils/utils_test.go [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+	"github.com/limechain/hedera-eth-bridge-validator/app/domain/service"
+	"github.com/limechain/hedera-eth-bridge-validator/app/router/response"
+	"github.com/limechain/hedera-eth-bridge-validator/test/mocks"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"strconv"
+	"testing"
+)
+
+var (
```
Comment (svetlio8) [inline app/router/utils/utils_test.go:34]: Maybe we can move them into `test/constants/`?

**Review by failfmi — DISMISSED** [review]

**Review by svetlio8 — DISMISSED** [review]

**Review by svetlio8 — APPROVED** [review]

**Pull request merged**